config/new_test.go

```diff
@@ -36,6 +37,8 @@ func TestNew_Options(t *testing.T) {
 		WithPortalURL("https://my-portal:1234"),
 		WithAPI("my-api:2345", &tls.Config{}),
 		WithSCADA("my-scada:3456", &tls.Config{}),
+		// considering that some fields now are optional...should that mean I should be passing in the Profile struct instance directly here (empty or otherwise)?
```

Comment:
I think the purpose of this test is to show what options there are, so personally I would populate the UserProfile with a project ID and org ID and then assert that the values are set correctly down below.